client/post-editor/editor-sharing/publicize-options.jsx

```diff
@@ -203,26 +166,6 @@ class EditorSharingPublicizeOptions extends React.Component {
 			);
 		}
 
-		if ( this.props.site && this.props.site.jetpack && ! this.props.isPublicizeEnabled ) {
```

Comment:
The condition a few lines above that checks for `this.props.site.options.publicize_permanently_disabled` is most likely dead, too. "publicize permanently disabled" implies "publicize disabled".

Reply:
Yeah, just tested disabling Publicize on a JP site and that message is not shown. 👍